cluster/charts/rook-ceph-cluster/values.yaml

```
@@ -0,0 +1,279 @@
+    #cleanup:
+
+  annotations:
+    #all:
+    #mon:
+    #osd:
+    #cleanup:
+    #prepareosd:
+    # If no mgr annotations are set, prometheus scrape annotations will be set by default.
+    #mgr:
+
+  labels:
+    #all:
+    #mon:
+    #osd:
+    #cleanup:
+    #mgr:
+    #prepareosd:
+    # monitoring is a list of key-value pairs. It is injected into all the monitoring resources created by operator.
+    # These labels can be passed as LabelSelector to Prometheus
+    #monitoring:
+
+  resources:
+    # The requests and limits set here, allow the mgr pod to use half of one CPU core and 1 gigabyte of memory
+    mgr:
+      limits:
+        cpu: "500m"
+        memory: "1024Mi"
+      requests:
+        cpu: "500m"
+        memory: "1024Mi"
+    # The above example requests/limits can also be added to the other components
+    #mon:
+    #osd:
+    #prepareosd:
+    #mgr-sidecar:
+    #crashcollector:
+    #logcollector:
+    #cleanup:
+
+  priorityClassNames:
+    all: #rook-ceph-default-priority-class
+    mon: #rook-ceph-mon-priority-class
+    osd: #rook-ceph-osd-priority-class
+    mgr: #rook-ceph-mgr-priority-class
+
+  # The section for configuring management of daemon disruptions during upgrade or fencing.
+  disruptionManagement:
+    # If true, the operator will create and manage PodDisruptionBudgets for OSD, Mon, RGW, and MDS daemons. OSD PDBs are managed dynamically
+    # via the strategy outlined in the [design](https://github.com/rook/rook/blob/master/design/ceph/ceph-managed-disruptionbudgets.md). The operator will
+    # block eviction of OSDs by default and unblock them safely when drains are detected.
+    managePodBudgets: true
+    # A duration in minutes that determines how long an entire failureDomain like `region/zone/host` will be held in `noout` (in addition to the
+    # default DOWN/OUT interval) when it is draining. This is only relevant when  `managePodBudgets` is `true`. The default value is `30` minutes.
+    osdMaintenanceTimeout: 30
+    # A duration in minutes that the operator will wait for the placement groups to become healthy (active+clean) after a drain was completed and OSDs came back up.
+    # Operator will continue with the next drain if the timeout exceeds. It only works if `managePodBudgets` is `true`.
+    # No values or 0 means that the operator will wait until the placement groups are healthy before unblocking the next drain.
+    pgHealthCheckTimeout: 0
+    # If true, the operator will create and manage MachineDisruptionBudgets to ensure OSDs are only fenced when the cluster is healthy.
+    # Only available on OpenShift.
+    manageMachineDisruptionBudgets: false
+    # Namespace in which to watch for the MachineDisruptionBudgets.
+    machineDisruptionBudgetNamespace: openshift-machine-api
+
+  # healthChecks
+  # Valid values for daemons are 'mon', 'osd', 'status'
+  healthCheck:
+    daemonHealth:
+      mon:
+        disabled: false
+        interval: 45s
+        timeout: ""
```

The default for mon failover is 10 minutes
```suggestion
        timeout: "10m"
```